### PR TITLE
feat: more files and badges

### PR DESCRIPTION
**Pull request by stevanlohja**
Just adding more assets and badges!

### CODE REVIEW

**Comment by stevanlohja** [comment]
Need at least 1 review to merge.

File: diagrams/generator-client/open-rpc-diagrams.xml [hunk 1]
```diff
@@ -0,0 +1 @@
...T17:50:59.167Z" host="www.draw.io" agent="Mozilla/5.0 (Macintosh; Intel Mac OS X 10_14_3) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/72.0.3626.121 Safari/537.36" etag="FEkksU15jwpALH7viu-s" version="10.5.2" type="device"><diagram id="f63et4fi7l6jguyWwufL" name="Page-1">7Vhdb9sgFP01fsxkTL76uKTZqqlTJ2VSn6l9Y7NiY2GcOPv1gxjHsWk+KqV1mu4pcIAA59x7OYmDp3HxXZA0+skDYI7nBoWDbx3PQ954pD40si6R0dAAoaCBmVQDc/oXDOgaNKcBZI2JknMmadoEfZ4k4MsGRoTgq+a0BWfNXVMSggXMfcJs9JEGMirR8cCt8TugYVTtjFwzEpNqsgGyiAR8tQPhmYOngnNZtuJiCkyTV/FSrvu2Z3R7MAGJPGXBr2hyjwp38hvf+tKNH+9+zJY9XH7LkrDcXNgcVq4rBiBQhJguFzLiIU8Im9XoRPA8CUBv46pePeee81SBSIF/QMq1UZfkkisokjEzo+WeeqO9dzNQxnPhw4ELVTFCRAjywDxvq4AKXeAxSLFW6wQwIumyeQ5iYijczqtpVg3D9CtYRxbrPIWkJ1Lf8YZMHXvyJFQr1K3MjyAmNr6ginsLtdRTUZfqZsKllmoVUQnzlGw4XKmUbQrxRPzncCPnQy4ZTcDgARHPD2oVlTqN3S/uQJ+slBONtvsuQUgoDmtoc24WjE2erKvSYfqrnbQzULSbcVXCnV2l4bXlhndibgy6zA3PYj2EBASRXPR8RvWl2yocCWqSpeXLsKCFVuIcwer1j0frNjLfJ1wHFnGXQ1ffuzi6xvviTLmN42VVUSGbtGVS8GeYcsbFptzq2qmLNGtBhNEwUV1fUQcKn2hiqTIcX81ATIOA7avVzXJyBmXwsKXM2Fam/4Iw3lvpcmPpkpIsu14BUPvd61oAZFfgD/7uIXziw4c6ffmQbcb/28K2LcS4a1uI+leXH4NT82PYaX7YDucjWMOXIvadvQ6yf8pcDl9tb3gJfI32RdrnNof4pmtvYrv2z+UO31AB1a3/jNyM7fyli2f/AA==</diagram></mxfile>
```
Comment (shanejonas) [inline diagrams/generator-client/open-rpc-diagrams.xml]: can we change the diagram to say `OpenRPC Document` instead of `open-rpc schema file`

Comment (stevanlohja) [reply diagrams/generator-client/open-rpc-diagrams.xml]: change has been made and in this PR now https://github.com/stevanlohja/design/blob/master/diagrams/generator-client/open-rpc-diagrams.png

**Review by shanejonas — APPROVED** [review]

**Comment by shanejonas** [comment]
thank-you

**Pull request merged**